activation/2.0/_index.md

```diff
@@ -32,20 +32,20 @@ summary: "Release for Jakarta EE 9"
 
 [Jakarta Activation 2.0 Release Record and Plan]()
```

Comment:
I know this wasn't part of your effort, but could you also include a link to the release record?  
https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0/plan

Thank you!